blueprint/core/blueprint_ReactApplicationRoot.h

```diff
@@ -102,66 +76,74 @@ namespace blueprint
             // We early return here in the event that we're currently showing the red error
             // screen. This prevents subsequent errors caused by dispatching events with an
             // incorrect engine state from overwriting the first error message.
-            if (errorText)
+            if (errorText != nullptr)
                 return;
 
-            try {
+            try
+            {
                 engine->invoke("__BlueprintNative__.dispatchViewEvent", std::forward<T>(args)...);
-            } catch (const EcmascriptEngine::Error& err) {
+            }
+            catch (const EcmascriptEngine::Error& err)
+            {
                 handleRuntimeError(err);
             }
         }
 
         //==============================================================================
         /** Displays the red error screen for the given error. */
-        void handleRuntimeError(const EcmascriptEngine::Error& err);
+        void handleRuntimeError (const EcmascriptEngine::Error& err);
 
         /** Clears the internal EcmascriptEngine and view table. */
         void reset();
 
         /** Installs the rendering hooks needed by the React reconciler into the
-         *  EcmascriptEngine environment.
-         *
-         *  Should be called after every `reset` and before evaluating the bundle.
-         */
+            EcmascriptEngine environment.
+
+            Should be called after every `reset` and before evaluating the bundle.
+        */
         void bindNativeRenderingHooks();
 
+        //==============================================================================
+        void resized() override;
+        void paint (juce::Graphics& g) override;
+
+       #if JUCE_DEBUG || DOXYGEN
+        /** In debug builds, we add a keypress handler to toggle debugging. */
+        bool keyPressed (const juce::KeyPress& key) override;
+       #endif
+
     private:
         //==============================================================================
-        template <int NumParams, typename MethodType>
+        ViewManager viewManager;
```

Comment:
Tried to keep members first / methods next in the private section. Doing that is much easier to visually scan.